src/int128_impl.h

```diff
@@ -0,0 +1,25 @@
+/***********************************************************************
+ * Copyright (c) 2014 Pieter Wuille                                    *
```

Comment:
Arguably this file is a derived work of your `scalar_impl.h` file.  But I'm happy to remove it if you like.

Reply:
Done.

Reply:
Not sure if it's better without a copyright header.  

> Arguably this file is a derived work of your `scalar_impl.h` file. :) But I'm happy to remove it if you like.

Then you may be violating the MIT license because it requires you to keep the copyright notice. :D


For me personally, I don't care. In the interest of getting this merged, I'll ACK this with and without notice, and with any of your names. But if you ask me, we should really see if I can make copyright notices easier in the future.